README.md

```diff
@@ -6,8 +6,8 @@ The `oVirt.disaster-recovery` role responsible to manage the disaster recovery s
 Requirements
 ------------
 
- * Ansible version 2.7.2 or higher
- * Python SDK version 4.2.4 or higher
+ * Ansible version 2.9 or higher
+ * Python SDK version 4.3 or higher
```

Comment:
Master development should aim only to 4.4

Reply:
Thanks, fixed :)